site/_includes/layouts/extension-samples.njk

```diff
@@ -0,0 +1,172 @@
+{% extends "layouts/base.njk" %}
+
+{% block css %}
+  {% InlineCss '/css/extension-samples.css' %}
+{% endblock %}
+
+{% block title_bar %}
+  {% set sharingEnabled = true %}
+  {% include 'partials/breadcrumbs.njk' %}
+{% endblock %}
+
+{% from 'macros/cards/hero-card.njk' import heroCard with context %}
+{% from 'macros/icon.njk' import icon with context %}
+
+{% set checkboxGroupI18n = {
+  reset: 'i18n.common.checkbox_group.reset' | i18n,
+  select_all: 'i18n.common.checkbox_group.select_all' | i18n
+} %}
+
+{% set typeLabel = 'i18n.extension_samples.type' | i18n %}
+{% set apiLabel = 'i18n.extension_samples.api' | i18n %}
+{% set permissionLabel = 'i18n.extension_samples.permission' | i18n %}
+
+{% block content %}
+  <div class="display-flex gap-top-300 lg:gap-top-400">
+    {% include 'partials/navigation-tree.njk' %}
+    <div class="extension-samples-container width-full pad-left-400 pad-right-400">
+      <div class="pad-bottom-400">
+        {{ heroCard(
+        title,
+        description,
+        'i18n.extension_samples.read_more' | i18n(locale),
```

Comment:
Thoughts on removing this button? I think we'll probably delete that page once your one is live.